docs/source/faq.rst

```diff
@@ -3,6 +3,13 @@ FAQ
 
 Here, we've compiled answers to commonly asked questions about metasyn and its development. If you have any other questions, need further assistance, or want to discuss something related to metasyn, don't hesitate to contact us directly. You can find our contact details on the :doc:`contact page </about/contact>`. We're more than happy to assist you and provide any additional information you may need.
 
+**Why did you change the name from MetaSynth to metasyn?**
+
+The project was originally named MetaSynth. However, as we progressed, we discovered that there was already an existing and established audio synthesis software under the same `name <https://uisoftware.com/metasynth/>`_. To avoid potential confusion between these two unrelated projects, we have decided to change the name of our project to **metasyn**. This new name still reflects the package's core goal, of metadata-driven data synthesis. We have also changed the styling of the name to be all lowercase to align with how the package is used in code (e.g. ```import metasyn```).
+
+It is important to note that despite the name change, metasyn as a project and the package's functionality remain the same. We apologize for any confusion caused by the renaming. 
```

Comment:
Personally, I don't think we need to apologize!

Reply:
I agree it's definitely not necessary, but I feel like it is a nice gesture that doesn't hurt. But I can remove it if you'd like?

Reply:
Personally, I feel like it is not necessary with an open source project. "We"'re not really an entity like a corporation would be. Of course for now we only have so many contributors, but maybe in the future there will be many more! (hopefully haha)

Reply:
That makes sense, changed!